client/src/components/DirectoryListing.vue

```diff
@@ -43,11 +75,25 @@ export default {
       resultSize.value = newResultSize;
     }
     watch(fileList, (currentValue, oldValue) => {
-      if (currentValue.length && currentValue.length > 0 && resultSize.value === 0) {
+      if (
+        currentValue.length &&
```

Comment:
Warum überprüfst du hier mit Bool-Konversion und mit einem Vergleichsoperator?
Ich glaube du kannst die erste Abfrage weglassen. undefined liefert in vergleichsoperatoren immer false
Aber not sure ob das das ist, was du wolltest

Reply:
optional :)